src/components/ImageCarousel.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import { Dimensions, StyleSheet } from 'react-native'
+import Carousel, { Pagination } from 'react-native-snap-carousel'
+import { ImagesType, ImageType } from '../constants/endpoints'
+import styled from 'styled-components/native'
+
+const { width: viewportWidth } = Dimensions.get('window')
+
+const StyledImage = styled.Image`
+  width: 100%;
+  height: 100%;
+  position: relative;
+`
+
+const styles = StyleSheet.create({
```

Comment:
How can I avoid the use of StyleSheet, if I don't want to use inline styles?

Reply:
Just pass `style={{ position: 'absolute', marginTop: -10, alignSelf: 'center' }}` as prop I guess :thinking:

Reply:
yeah, that's inline style :D

Reply:
ups yeah I see :D no idea than :man_shrugging: